v4l2/v4l2_decode.cpp

```diff
@@ -330,7 +330,9 @@ int32_t V4l2Decoder::ioctl(int command, void* arg)
         ASSERT(!m_streamOn[INPUT] && !m_streamOn[OUTPUT]);
         if (format->type == V4L2_BUF_TYPE_VIDEO_CAPTURE_MPLANE) {
             // ::Initialize
-            ASSERT(format->fmt.pix_mp.pixelformat == V4L2_PIX_FMT_NV12M);
+            memcpy(&m_configBuffer.size, format->fmt.raw_data, sizeof(uint32_t));
```

Comment:
better check the size, is it larger than sizeof(format.fmt.raw_data)- sizeof(uint32_t)?